packages/react-graphql-universal-provider/src/test/request-id-link.test.ts

```diff
@@ -0,0 +1,25 @@
+import gql from 'graphql-tag';
+import {ApolloLink, Observable} from 'apollo-link';
+
+import {createRequestIdLink} from '../request-id-link';
+
+const query = gql`
```

Comment:
I don't see this referenced anywhere, do we need it?

Reply:
🔥 ing!